extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js

```diff
@@ -47,6 +48,19 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
     return true;
   }
 
+  /** @override */
+  buildCallback() {
+    return this.mutateElement(() => {
+      setStyles(this.element, {
+        'position': 'absolute',
+        'width': '1px',
+        'height': '1px',
+        'overflow': 'hidden',
+        'visibility': 'hidden',
+      });
+    });
```

Comment:
Does this result in a flash of a visible element before build?

Reply:
Nope, because we are enforcing `layout="nodisplay"` on the element 😄 Thanks for checking on this though!